src/main.cpp

```diff
@@ -416,7 +425,7 @@ int StartClient(const ClientConfig &config) {
                  config.peer_ip, BIND_PORT);
 
 
-    int tap_fd = CreateTapDevice(config.bridge_ip);
+    int tap_fd = CreateTapDevice(config.bridge_ip, "255.255.255.0");
```

Comment:
Should we make this dynamic?
E.g. have the user provide the bridge IP like `172.16.78.2/24` (/24 being the netmask?)

Reply:
I was just too lazy for that and didn't know whether it's necessary. So - maybe merge in the current state and keep it as "feature request" in case someone actually needs it?